h2o-core/src/main/java/hex/faulttolerance/Recovery.java

```diff
@@ -13,8 +19,30 @@
 import java.util.*;
 
 public class Recovery<T extends Keyed> {
```

Comment:
This is a main class of its domain, a little JavaDoc with explanation of what it does, how it should be used and why won't hurt.

Reply:
good idea, done